### ISSUE TITLE
404 page

### ISSUE THREAD

**Issue by postoroniy** [JuliaImages/ImageFeatures.jl#22]
page is not found 
https://juliaimages.github.io/ImageFeatures.jl/stable

**Comment by timholy** [comment]
Good point. I don't think the stable docs get generated until the package has a tagged release. @mronian, what are your thoughts on registering the package? Or we could delete the link for now.

**Comment by mronian** [comment]
I think registering the package would help attract more users. We already have docs and tests for the code. Is anything else required before registering the package?

**Comment by timholy** [comment]
I don't think so. We might want to see what needs updating to the new Images. Do you want me to do that, or would you like to work through it yourself? (Either is fine with me.)

**Comment by mronian** [comment]
I'll need some time to get familiar with the changes in Images.jl. I'll start updating ImageFeatures by next week :)

**Comment by Enet4** [comment]
Any updates on this? The library as a whole looks rather useful, but it's hard and risky to pick it up without a registered package or a proper readme.

**Comment by timholy** [comment]
Just replace "stable" with "latest" in the URL. But really we should register this. @mronian, I seem to remember you wanted #21 first?
